regression/pages/whitelabel/const.py

```diff
@@ -0,0 +1,327 @@
+    u'Venezuela', u'Vietnam',
+    u'Virgin Islands (British)', u'Virgin Islands (U.S.)',
+    u'Wallis and Futuna', u'Western Sahara', u'Yemen',
+    u'Zambia', u'Zimbabwe'
+]
+
+COUNTRIES = [x.replace(' ', '') for x in COUNTRIES_WITH_SPACES]
+
+LANGUAGES_WITH_SPACES = [
+    u'', u'Afar', u'Abkhazian', u'Afrikaans', u'Akan', u'Albanian',
+    u'Amharic', u'Arabic', u'Aragonese',
+    u'Armenian', u'Assamese', u'Avaric', u'Avestan', u'Aymara',
+    u'Azerbaijani', u'Bashkir', u'Bambara',
+    u'Basque', u'Belarusian', u'Bengali', u'Bihari languages',
+    u'Bislama', u'Bosnian', u'Breton',
+    u'Bulgarian', u'Burmese', u'Catalan', u'Chamorro', u'Chechen',
+    u'Chinese', u'Simplified Chinese',
+    u'Traditional Chinese', u'Church Slavic', u'Chuvash', u'Cornish',
+    u'Corsican', u'Cree', u'Czech',
+    u'Danish', u'Divehi', u'Dutch', u'Dzongkha', u'English',
+    u'Esperanto', u'Estonian', u'Ewe', u'Faroese',
+    u'Fijian', u'Finnish', u'French', u'Western Frisian',
+    u'Fulah', u'Georgian', u'German', u'Gaelic',
+    u'Irish', u'Galician', u'Manx', u'Greek', u'Guarani',
+    u'Gujarati', u'Haitian', u'Hausa', u'Hebrew',
+    u'Herero', u'Hindi', u'Hiri Motu', u'Croatian', u'Hungarian',
+    u'Igbo', u'Icelandic', u'Ido', u'Sichuan Yi',
+    u'Inuktitut', u'Interlingue', u'Interlingua', u'Indonesian',
+    u'Inupiaq', u'Italian', u'Javanese',
+    u'Japanese', u'Kalaallisut', u'Kannada', u'Kashmiri', u'Kanuri',
+    u'Kazakh', u'Central Khmer', u'Kikuyu',
+    u'Kinyarwanda', u'Kirghiz', u'Komi', u'Kongo', u'Korean',
+    u'Kuanyama', u'Kurdish', u'Lao', u'Latin',
+    u'Latvian', u'Limburgan', u'Lingala', u'Lithuanian',
+    u'Luxembourgish', u'Luba-Katanga', u'Ganda',
+    u'Macedonian', u'Marshallese', u'Malayalam', u'Maori',
+    u'Marathi', u'Malay', u'Malagasy', u'Maltese',
+    u'Mongolian', u'Nauru', u'Navajo', u'Ndebele, South',
+    u'Ndebele, North', u'Ndonga', u'Nepali',
+    u'Norwegian Nynorsk', u'Bokm\xe5l, Norwegian', u'Norwegian',
+    u'Chichewa', u'Occitan', u'Ojibwa', u'Oriya',
+    u'Oromo', u'Ossetian', u'Panjabi', u'Persian', u'Pali',
+    u'Polish', u'Portuguese', u'Pushto', u'Quechua',
+    u'Romansh', u'Romanian', u'Rundi', u'Russian', u'Sango',
+    u'Sanskrit', u'Sinhala', u'Slovak', u'Slovenian',
+    u'Northern Sami', u'Samoan', u'Shona', u'Sindhi', u'Somali',
+    u'Sotho, Southern', u'Spanish', u'Sardinian',
+    u'Serbian', u'Swati', u'Sundanese', u'Swahili', u'Swedish',
+    u'Tahitian', u'Tamil', u'Tatar', u'Telugu',
+    u'Tajik', u'Tagalog', u'Thai', u'Tibetan', u'Tigrinya',
+    u'Tonga (Tonga Islands)', u'Tswana', u'Tsonga',
+    u'Turkmen', u'Turkish', u'Twi', u'Uighur', u'Ukrainian',
+    u'Urdu', u'Uzbek', u'Venda', u'Vietnamese',
+    u'Volap\xfck', u'Welsh', u'Walloon', u'Wolof', u'Xhosa',
+    u'Yiddish', u'Yoruba', u'Zhuang', u'Zulu'
+]
+
+LANGUAGES = [x.replace(' ', '') for x in LANGUAGES_WITH_SPACES]
+
+# Default Threshold for visual difference
+DIFF_THRESHOLD = 5000
```

Comment:
I see the point but think this is not a smart approach. The diff threshold is based on number of pixels. So, you could have a very small image that is a 0% match, but if it's less than 5000 in size, it will always pass.